routes/web.php

```diff
@@ -76,6 +76,8 @@
 
     Route::get('/', [RegisterController::class, 'showRegistrationForm'])->name('register')->middleware(OnlyHungarian::class);
     Route::get('/guest', [RegisterController::class, 'showTenantRegistrationForm'])->name('register.guest');
+
+    Route::post('/rerequest_tenant_status', [RegisterController::class, 'rerequestTenantStatus'])->name('register.rerequest_tenant_status');
```

Comment:
undefined function

Reply:
Yepp; I've forgot about that. It was a remnant of an earlier approach.